src/Apps/Order/Routes/Shipping2/Components/AddressModal2.tsx

```diff
@@ -30,48 +30,12 @@ import { useUpdateSavedAddress } from "Apps/Order/Routes/Shipping2/Mutations/use
 import { useUpdateUserDefaultAddress } from "Apps/Order/Routes/Shipping2/Mutations/useUpdateUserDefaultAddress"
 import { useShippingContext } from "Apps/Order/Routes/Shipping2/Hooks/useShippingContext"
 
-export enum AddressModalActionType {
```

Comment:
This just moves less relevant file details to undernieth the main component export

Reply:
That's a good idea I was curious about what determined this